service/src/main/resources/api/service_openapi.yaml

```diff
@@ -1432,6 +1432,12 @@ components:
           type: string
           description: Optional display string
           nullable: true
+        repeatedValue:
+          type: array
+          items:
+            $ref: "#/components/schemas/Literal"
+        isRepeatedValue:
```

Comment:
What is this for? Doesn't the existence of the `repeatedValue` (or it being non-empty) already indicate this?

Reply:
This is to distinguish an empty array from a non-repeated/scalar value. Some of the rows in the variant table have empty arrays.

Reply:
Will there potentially be other types as well and this should be an enum? Should `value` be deprecated and replaced with `values` instead?

Reply:
I don't have plans for any other types, though if they come up then an enum certainly sounds like a good choice to consider.

I'm happy to remove `value` here and just always populate a single element of `values` instead -- would that be easier for you? I didn't do it initially because it wouldn't be backwards compatible, so I figured you'd need to make a bunch of changes on the frontend to accommodate.

Reply:
Per offline discussion, we decided to:
- Always populate the values array, so the UI can move over to always using that, without breaking backwards compatibility immediately.
- Continue populating the is repeated flag, just in case the UI hits a case where it needs to distinguish between a single element array and a non-repeated value. If it turns out the UI doesn't need to distinguish between those two cases, we can remove this flag from OpenAPI in the future.